source/reference/command/setAuditConfig.txt

```diff
@@ -0,0 +1,204 @@
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. dbcommand:: setAuditConfig
+
+   .. versionadded:: 5.0
+
+   :dbcommand:`setAuditConfig` is an administrative command that
+   sets new audit configurations for :binary:`~bin.mongod` and
+   :binary:`~bin.mongos` server instances at runtime.
+
+   The command syntax is:
+
+   .. code-block:: javascript
+
+      { setAuditConfig: 1, filter: <Filter Document>, auditAuthorizationSuccess: <Boolean> }
+
+   :dbcommand:`setAuditConfig` has the following fields:
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 20 20 80
+
+      * - Field
+        - Type
+        - Description
+
+      * - ``setAuditConfig``
+        - integer
+        - Run :dbcommand:`setAuditConfig`
+
+      * - ``filter``
+        - document
+        - An :doc:`audit filter </tutorial/configure-audit-filters>`
+
+      * - ``auditAuthorizationSuccess``
+        - boolean
+        - Log all, or only failed access authorizations
+
+   Use the
+   :method:`db.adminCommand( { command } )<db.adminCommand()>` method
+   to run :dbcommand:`setAuditConfig` against the ``admin`` database.
+
+Behavior
+--------
+
+Enable :doc:`Auditing </core/auditing>` to use
+:dbcommand:`setAuditConfig` at runtime.
+
+:parameter:`auditAuthorizationSuccess` enables
+:doc:`auditing </core/auditing>` of authorization success for the
+:ref:`authCheck <audit-action-details-results>` action. The parameter
+value must be ``true`` to audit :authaction:`read` and
+:authaction:`write` operations. However, when
+:parameter:`auditAuthorizationSuccess` is ``false`` auditing has less
+performance impact because the :doc:`audit system </core/auditing>`
+only logs authorization failures.
+
+Configuration updates are distributed via the :term:`oplog` mechanism
+which means updates on :binary:`~bin.mongod` nodes are distributed to
+secondary nodes very quickly. There is a different distribution
+mechanism on :binary:`~bin.mongos` nodes. :binary:`~bin.mongos` nodes
+have to :parameter:`poll <auditConfigPollingFrequencySecs>` the primary
+server at regular intervals for configuration updates. You may see
+stale data due to polling delay if you run :dbcommand:`setAuditConfig`
+on the primary server and :dbcommand:`getAuditConfig` on a
+:doc:`shard </sharding>` before the shard has polled the primary server
+for updated configuration details.
+
+Examples
+--------
+In these examples the audit messages have been reformatted. They appear
+on a single line in the log file.
+
+Audit Collection Creation and Deletion
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enable auditing when a collection is created or deleted.
+
+.. code-block:: javascript
+
+   db.admin.runCommand(
+      {
+         setAuditConfig: 1,
+         filter:
+            {
+               atype:
+                  {
+                     $in:
+                        [ "createCollection", "dropCollection" ]
```

Comment:
These should be on the same line.

`$in: [ "createCollection", "dropCollection" ]`